### PR TITLE
RD- 2933 Update Dashboard page

### PR DESCRIPTION
**Pull request by qooban**
## Description

1. Updated all the links to examples to point to the latest version - v6.2
2. Updated Dashboard page to show new widgets (see mockup in RD-2698)
3. Adjusted some widgets to be aligned with the rest (see comments in the code)
4. Remove showing tabs in case only one Blueprint Marketplace tab is defined

## Screenshots / Videos

https://user-images.githubusercontent.com/5202105/131962744-0fadeafb-652d-4363-8f4b-55c311238273.mp4

NOTE: I used another filter for K8S, but kubernetes cluster label (`csys-k8s-filter`) is going to be added to Cloudify Manager (see RD-2998).

## Checklist

- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests
1. [Stage-UI-System-Test/1186](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1186/pipeline) (2021-09-02 12:39:32 CEST) - PASSED
2. [Stage-UI-System-Test/1190](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1190/pipeline) (2021-09-03 08:37:47 CEST) - PASSED
3. [Stage-UI-System-Test/1211](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1211/pipeline) (2021-09-07 10:40:55 CEST) - PASSED

I'm also planning to add some tests for Dashboard page once we get final approval for the look'n'feel. Possibly in a separate PR.

## Documentation
https://docs.cloudify.co/latest/working_with/console/pages/dashboard-page/ needs to be updated. There will be another PR for that once we get this PR merged.

### CODE REVIEW

File: test/cypress/integration/widgets/blueprint_catalog_spec.ts [hunk 1]
```diff
@@ -3,9 +3,9 @@ describe('Blueprints catalog widget', () => {
 
     before(() =>
         cy
-            .activate('valid_trial_license')
```
Comment (qooban) [inline test/cypress/integration/widgets/blueprint_catalog_spec.ts:6]: 'valid_trial_license' is set by default, so I decided to remove it from here.

File: widgets/deploymentNum/src/widget.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ Stage.defineWidget<unknown, WidgetData, DeploymentNumWidgetConfiguration>({
     description: t('description'),
     initialWidth: 2,
     initialHeight: 8,
-    color: 'green',
+    color: 'blue',
```
Comment (qooban) [inline widgets/deploymentNum/src/widget.tsx:29]: To keep it aligned with "Number of Clouds" widget, I decided to change the color to blue.

File: widgets/serviceButton/src/ServiceButton.tsx [hunk 3]
```diff
@@ -21,8 +21,9 @@ const ServiceButton: FunctionComponent<ServiceButtonProps> = ({ basic, color, ic
             <Button
                 basic={basic}
                 color={color || undefined}
-                icon={icon || undefined}
                 content={label}
+                icon={icon || undefined}
+                fluid
```
Comment (qooban) [inline widgets/serviceButton/src/ServiceButton.tsx:26]: To use full width of the widget box and keep it aligned with "Button Link" widget I decided to go with the fluid button.

**Review by kubama — DISMISSED** [review]
Thanks for adding in-code comments, it made the review a much nicer experience 👍.

**Review by stepek — APPROVED** [review]
code lgtm

**Comment by qooban** [comment]
Just one small change added - application package size limit increase: https://github.com/cloudify-cosmo/cloudify-stage/pull/1576/commits/8d839be91fe98aefb3282603c83894831a420a7c .

As stated in the e-mail:

> Looking at the plans for 6.3 I'm not going to add system tests for the new dashboard, also I'm not planning to document my changes.

Final system tests run: [Stage-UI-System-Test/1211](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1211/pipeline) (2021-09-07 10:40:55 CEST).

Once I get your final approval, I'll merge it to `master` and then cherry-pick onto `6.2.0-build` branch.

**Review by kubama — APPROVED** [review]

**Review by stepek — DISMISSED** [review]

**Comment by qooban** [comment]
I updated the branch with latest `master` and resolved the conflict. Having two approvals before, I'll consider single approval as "GO" for merge.

**Review by stepek — APPROVED** [review]

**Review by kubama — APPROVED** [review]

**Pull request merged**